utils/src/config.rs

```diff
@@ -136,25 +119,24 @@ pub struct ServerTimeout {
     #[serde(with = "duration_format", default = "default_rpc_timeout")]
     rpc_timeout: Duration,
 
-    /// How long a candidate should wait before it starts another round of election
+    /// How many ticks a candidate needs to wait before it starts a new round of election
+    /// It should be larger than `follower_timeout`
```

Comment:
According to the blog(https://microraft.io/blog/2021-09-08-today-a-raft-follower-tomorrow-a-raft-leader/), `candidate_timeout_ticks` should be less than `follower_timeout_ticks`